linux/Jenkinsfile

```diff
@@ -338,10 +389,34 @@ def uploadDebArtifacts(String buildArch) {
         } else {
             env.BUILDARCH = debArchList[buildArch]
             env.DISTROLIST = distro_list
-            jf 'rt u **/build/ospackage/temurin-*${BUILDARCH}.deb deb/pool/main/t/temurin-${VERSION}/ --target-props=${DISTROLIST}deb.component=main;deb.architecture=${BUILDARCH} --flat=true'
+
+            def PackFiles = findFiles(glob: "**/build/ospackage/temurin-*${debArchList[buildArch]}.deb") // List All Packages To Upload
+            for (PackFile in PackFiles) {
+              def FileName = PackFile.name
```

Comment:
this logic is similar to the APJ logic, worth exrtracting?

Reply:
I did consider this, but given the globs, targets and upload commands are different for APK, DEB & RPM, the only element of abstraction would be the curl check, and not worth the added complexity. As Im hoping to further enhance this process in the near future, I think this is sufficient to solve the current problems we encounter at release time.

Reply:
Reusable function added.